.devcontainer/devcontainer.json

```
@@ -0,0 +1,33 @@
+{
+	"name": "Documentation Container",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": ".."
+	},
+
+	"features": {
+		"ghcr.io/devcontainers/features/common-utils:2": {
+			"username": "automatic",
+			"uid": "automatic",
+			"gid": "automatic",
+			"installZsh": false,
+			"installOhMyZsh": false,
+			"upgradePackages": false,
+			"nonFreePackages": false
+		}
+	},
+    "customizations": {
+		"vscode": {
+			"extensions": [
+				"ms-azuretools.vscode-docker",
+				"github.vscode-github-actions",
+				"ms-vscode.cpptools-extension-pack",
+				"GitHub.copilot",
+                "ms-vscode.cpptools",
+                "ms-vscode.cmake-tools"
+			]
+		}
+	},
+    "workspaceFolder": "/home/app/",
+    "workspaceMount": "source=${localWorkspaceFolder},target=/home/app/,type=bind,consistency=cached"
+}
```

```suggestion
}

```
End file with a new line.